src/applications/ivc-champva/10-7959C/sass/10-7959C.scss

```diff
@@ -7,5 +7,10 @@
 @import "../../../../platform/forms/sass/m-form-confirmation";
 
 .upload-button-label {
-    margin-top: 8px;
-}
\ No newline at end of file
+  margin-top: 8px;
+}
+
+va-accordion-item[data-chapter="fileUpload"] {
+  display: none;
+  visibility: hidden;
+}
```

Comment:
Using CSS fix because despite all the individual pages being set to not show on review, the chapter step in the accordion still does (presumably because this is an edge case)